Storage/SQL/SQLiteHistory.swift

```diff
@@ -299,9 +299,7 @@ extension SQLiteHistory: BrowserHistory {
         let (query, args) = self.filteredSitesByFrecencyQueryWithHistoryLimit(limit, bookmarksLimit: 0, groupClause: groupBy, whereData: whereData)
         let cacheArgs: Args = [limit]
 
-        return accumulate([
-            { self.db.runQuery(query, args: args, factory: SQLiteHistory.iconHistoryColumnFactory) },
```

Comment:
A similar thing was done here. Code here was split into new lines to make them more readable.